pages/posts/the-last-breaking-change.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "The Last Breaking Change"
```

Comment:
I think this is promising too much at this point. We've decided that this one breaking change needs to happen, not that there won't be others in the next release.

Reply:
I can add some text that says there may be other, less impactful breaking changes, but I do want to highlight this one as forcing our hand.

Reply:
I noticed that the text acknowledges that there could be other changes, but I think the title contradicts that text. There must be a way to highlight that this change is forcing our hand without implying that it's the only change we might choose (or need) to make for the next release.

Reply:
The intent of the title isn't identifying each breaking change individually, but looking at the spec as a whole breaking things for the last time.

Reply:
I added another section toward the end to discuss the possibility of including other changes.

Reply:
Ahh, I see what you were trying to do now. But, if I got the wrong idea, others probably will too. I feel like it would be more accurate if it said "The Last Release With Breaking Changes", but that's pretty clunky.